File: Chapter 05/C Code/5_11.c
#include <stdio.h>

int main() {
    int a[3][3] = {{1,1,1},{0,1,1},{0,0,1}};
    int flag = 0;
    
    for (int i = 0; i < 3; i++) {
        for (int j = 0; j < i; j++) {
            if (a[i][j] !=a[j][i]) {
                flag = 1;
                break;
            }
        }
        if (flag == 1) {
            break;
        }
    }
    
    if (flag == 0) {
        printf("Matrix is symmetric \n");
    } else {
        printf("Matrix is not symmetric \n");
    }
    
    return 0;
}

File: Chapter 05/Python code/5_11.py
#a=[[1,1,1],[1,1,1],[1,1,1]]
a=[[1, 2, 3], [2, 3, 4], [3, 4, 6]]

c=[]
for i in range(0, len(a[0])):
    t=[]
    for j in range(0, len(a)):
        t.append(a[j][i])
    c.append(t)
    del t
print("Original Matrix",a)
print("Transposed Matrix",c)

if a==c:
    print("Symmetric")
else:
    print("not Symmetric")
